install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/templates/mutatingwebhookconfiguration.yaml.tpl

```diff
@@ -1,3 +1,4 @@
+{{ define "mutatingwebhookconfiguration.yaml.tpl" }}
```

Comment:
If we've got a custom version of this file in Maistra/istio, can we integrate all of the updates here (i.e. the changes being made by the sed statements in update-yaml and patch-charts.sh)?

Reply:
I'd prefer not, since most of the changes you refer to are OpenShift specific ones or related to upstream changes we're not involved with.  I intend this to go upstream soon so we are pulling in a change we expect from a future upstream release, we've done this a couple of times in the past but not very often.

Reply:
The one thing that I would have recommended was removing the namespace field from this, since these resources are non-namespaced.

Reply:
The namespace is coming from upstream, while superfluous it doesn't have any bearing on the configuration and doesn't prevent the code from working so it's not necessary for us to remove it.